Assets/Scripts/Functional Definitions/Abilities/Disrupt.cs

```diff
@@ -40,12 +40,27 @@ protected override void Execute()
                         if(part.GetComponent<Ability>() && !(part.GetComponent<Ability>() as PassiveAbility)){
                             //part.SetPartColor(Color.grey);
                             part.lerpColors();
-                        }
+                            var missileLinePrefab = new GameObject("Missile Line"); // create prefab and set to parent
+                            missileLinePrefab.transform.SetParent(transform, false);
+
+                            var missileColor = part && part.info.shiny ? FactionManager.GetFactionShinyColor(Core.faction) : new Color(0.8F, 1F, 1F, 0.9F);
 
+                            // I use this prefab as one of the active lines on the missile 
+                            // because what's the point in not doing it this way
+
+                            LineRenderer lineRenderer = missileLinePrefab.AddComponent<LineRenderer>(); // add line renderer
+                            lineRenderer.material = ResourceManager.GetAsset<Material>("white_material"); // get material
+                            MissileAnimationScript comp = missileLinePrefab.AddComponent<MissileAnimationScript>(); // add the animation script
+            
+                            var x = Instantiate(missileLinePrefab, part.transform); // instantiate
+                            x.GetComponent<MissileAnimationScript>().Initialize(); // initialize
+                            x.GetComponent<MissileAnimationScript>().lineColor = missileColor;
+                            Destroy(x, 0.5f);
+                            base.Execute();
```

Comment:
This would be executing the base function multiple times, just return it to how it was before

Reply:
done